### PR TITLE
Formatting of an (old) ChIP-seq tutorial

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

With this PR, the second ChIP-seq tutorial is formatted to the template. 

@erxleben, @bgruening Can you check the content?

To do

- [X] Extract the workflows
- [X] Extract the the tools (from the workflows)
- [x] Upload data on Zenodo
- [x] Fill the data library file
- [ ] Add a data manager file?
- [ ] Add an Interactive tour

Bérénice

### CODE REVIEW

**Comment by bgruening** [comment]
Ok, I added a few comments, this should be in a merge able state maybe. We should revise this in a few weeks if the storm has passed.

File: topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+>    - [`patient1_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_ChIP_ER_good_outcome.bam)
+>    - [`patient1_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_input_good_outcome.bam)
+>    - [`patient2_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_ChIP_ER_good_outcome.bam)
+>    - [`patient2_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_input_good_outcome.bam)
+>    - [`patient3_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_ChIP_ER_poor_outcome.bam)
+>    - [`patient3_input_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_input_poor_outcome.bam)
+>    - [`patient4_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient4_ChIP_ER_poor_outcome.bam)
+>    - [`patient4_input_poor_outcome.bam`](url: https://zenodo.org/record/892432/files/patient4_input_poor_outcome.bam)
+>
+> 3. **multiBamSummary** {% icon tool %} with
```
Comment (fidelram) [inline topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md]: Can you add here some extra info about deepTools in general and then on multiBamSummary? Something along this lines:

3. To compute the correlation between the samples we are going to to use the QC modules of deepTools (http://deeptools.readthedocs.io/), a software package for the QC, processing and analysis of NGS data. Before computing the correlation a time consuming step is required, which is to compute the read coverage over a large number of regions from each BAM file. For this we will use the tool **multiBamSummary**.

File: topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+>       This corresponds to the length of the fragments that were sequenced; it is not the read length!
+>
+>    - "Distance between bins" to `500000` (to reduce the computation time for the tutorial)
+>    - "Region of the genome to limit the operation to" to `chr1` (to reduce the computation time for the tutorial)
+>  
+>    This tool splits the reference genome into bins of equal size (*e.g.* 10kb) and counts the number of overlapping reads from each sample.
+>
+> 4. **plotCorrelation** {% icon tool %} with
+>    - "Matrix file from the multiBamSummary tool" to the generated multiBamSummary output
+>
```
Comment (fidelram) [inline topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md]: Here, I will also like to add some little background.

4. To compute and visualize the sample correlation we use **plotCorrelation** from deepTools. This is a fast process that allows the user to quickly try different color combinations and outputs.

File: topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+>    - "Choose computation mode" to `Bins`
+>    - "Bin size in bp" to `100`
+>       
+>       This corresponds to the length of the fragments that were sequenced; it is not the read length!
+>
+>    - "Distance between bins" to `500000` (to reduce the computation time for the tutorial)
+>    - "Region of the genome to limit the operation to" to `chr1` (to reduce the computation time for the tutorial)
+>  
+>    This tool splits the reference genome into bins of equal size (*e.g.* 10kb) and counts the number of overlapping reads from each sample.
+>
```
Comment (fidelram) [inline topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md]: Please replace to:
"Using these parameters, the tool will take bins of 100 bp separated by 500,000. For each bin the overlapping reads in each sample will be computed"

File: topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,531 @@
...
+>    <details>
+>    <summary>Click to view answers</summary>
+>       The difference between input and ChIP signal is not totally clear. 20% of chromosome 1 are not sequenced
+>       at all and the ChIP signal is only slightly more enrichted then the input.
+>    </details>
+{: .question}
+
+> ### {% icon hands_on %} (Optional) Hands-on: GC bias assessment (other samples)
+>
```
Comment (fidelram) [inline topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md]: I think this optional task should be for the fingerprint and not the GC bias.

**Comment by bebatut** [comment]
ping @bgruening

**Comment by bgruening** [comment]
these failing tests are strange ...

**Pull request merged**